src/App.js

```diff
@@ -5,18 +5,19 @@ import TodoList from './TodoList';
 
 function App() {
 
-  const [newTodo, setNewTodo] = React.useState('');
+  const [todoList, setTodoList] = React.useState([]);
+
+ function addTodo (newTodo) {
+    setTodoList([...todoList, newTodo]);
```

Comment:
No need to change it, but as an example that matches what the curriculum demonstrated you can also do it this way and it does the same thing. And yes `prevTodos` is a booger, it's typically either `prevState` or `prevWhateverYourStateIs`
```suggestion
    setTodoList((prevTodos) => [...prevTodos, newTodo]);
```